js/ui/projectManager.js

```diff
@@ -129,9 +129,18 @@ define([
     this.expanded = {};
     this.project = null;
     this.projectFile = null;
+
     if (element) {
       this.setElement(element);
+
+      // use host document object to bind resize events
+      if (document && !document.boundSidebarRersizeEvents) {
```

Comment:
I'm not wild about keeping the flag on the document (and it would need to be spelled correctly). I'm pretty sure we only call the constructor for the project manager once anyway, so we shouldn't need to guard against multiple bindings.